content/en/docs/components/pipelines/v2/run-a-pipeline.md

```diff
@@ -4,77 +4,101 @@ description = "Execute a pipeline on the KFP backend"
 weight = 6
```

Comment:
Should this be `title = "Run a pipeline"` to be consistent with "Compile a pipeline"?

Reply:
Assuming that the mention of "backend" is not essential, accepting the comment.

Reply:
Apologies if this was unclear. This wasn't a comment about the subtitle. I like it as it was before.

This was just whether the "p" of "pipeline" in "Run a Pipeline" should be lowercase to be consistent with the other change you suggested here: https://github.com/kubeflow/website/pull/3393/files#diff-f18c20da363f8a67e27396a3afab90fc9f260eebb2571e9473526bba8c10763eL2

Reply:
That makes sense. Fixed it.